docs/core/tools/dotnet-build.md

```diff
@@ -107,6 +109,14 @@ Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal
 
 Defines the version suffix for an asterisk (`*`) in the version field of the project file. The format follows NuGet's version guidelines.
 
+`-p|-property:<name>=<value>`
+
+Set or override a specific project-level property, where `name` is the property anme and `value` is the property value. See [MSBuild Command-Line Reference](/visualstudio/msbuild/msbuild-command-line-reference) for more information and other MSBuild supported switches.
+
+`-l|-logger:<logger>`
+
+Specifies the logger to use to log events from MSBuild. See [MSBuild Command-Line Reference](/visualstudio/msbuild/msbuild-command-line-reference) for more information and other MSBuild supported switches.
+
```

Comment:
A question: why those two options out of the full list of the MSBuild options got such an attention. Basically, the same remark as to the synopsis section update.